hydra-cluster/src/CardanoClient.hs

```diff
@@ -111,25 +111,22 @@ waitForUTxO node utxo =
     txOut ->
       error $ "Unexpected TxOut " <> show txOut
 
-mkGenesisTx ::
+mkInitialTx ::
   NetworkId ->
   -- | Owner of the 'initialFund'.
   SigningKey PaymentKey ->
   -- | Amount of initialFunds
   Coin ->
   -- | Recipients and amounts to pay in this transaction.
   [(VerificationKey PaymentKey, Coin)] ->
+  TxIn ->
+  -- | Initial input from which to spend
```

Comment:
Haddock referencing wrong parameter. Also, it would be more logical to put this after `NetworkId` and before other arguments which refer the "things to spend", while the "things to produce" come after